test/library/packages/Curl/check-http-setopt.chpl

```diff
@@ -52,7 +53,28 @@ proc test2() {
   stdout.flush();
 }
 
+// Test a string-type option, CURLOPT_URL
```

Comment:
Ok, done.

FYI, I tried making the test do

```chapel
proc main() {
  startServer();
  defer stopServer();
  test1();
  test2();
  test3();
  test4();
}
```

But when I had `test4()` pass `(url:bytes, )`, it threw an exception but the server wasn't killed on the way out.

```
...
uncaught SystemError: Invalid argument (in Curl.setopt(10002, arg: 1*bytes))
  $CHPL_HOME/modules/packages/Curl.chpl:203: thrown here
  check-http-setopt.chpl:83: uncaught here

$ ps auwx | grep http.server
fortytwo  147966  0.2  0.0 110968 17344 pts/7    S    08:39   0:00 python3 -m http.server 8000 --bind 127.0.0.1
```

Reply:
Right, that's because you are using "prototype error handling" which halts. If you put the whole program into a `module CheckHttpSetopt { ... }` then the compiler will insist some of these are decorated with `throws` and it should work.

Reply:
Ah, that makes sense, thanks.  I'd forgotten how that worked.  I can add that in a separate PR.